.github/workflows/build_and_test.yml

```diff
@@ -11481,7 +11481,7 @@ jobs:
         call setenv.cmd
         mkdir tests\cmake\build
         cd tests\cmake\build
-        cmake ..
+        cmake .. -DCMAKE_CXX_STANDARD=20
```

Comment:
`configure` probably needs to write `OPENDDS_CXX_STD` to `config.cmake` when `--std` is used.

Reply:
It would have to be translated though. I've mentioned this before, but it also doesn't help in the case where MPC/ACE/TAO is setting the standard internally.

Reply:
I went ahead and created it: #4638